Adding tooltip

scrubskip opened the pull request:
Added a JQuery tooltip which displays the current time, with seconds.

scrubskip committed:
Add local time formatting function.

scrubskip committed:
Add tooltip to slider.

bretlorimore reviewed (changes requested):
Justin, the slight bug you mention can be fixed as follows:

- insert `  $(this.slider).tooltip('disable');` at the end of 
  `EclipseSimulator.View.prototype.start_playing`
- insert `  $(this.slider).tooltip('enable');` at the end of the second if-statement in 
  `EclipseSimulator.View.prototype.stop_playing` (after 
  `$(this.playbutton).find('i').text(EclipseSimulator.PLAY_PAUSE_BUTTON[!this.playing]);`)
- Remove the mouseout event handler (lines 353-358) as it is no longer necessary. 

Also, if you could move the definition of `EclipseSimulator.get_local_time_from_date` to within the `EclipseSimulator` object the way that `compute_sun_moon_sep` is, that would be great so that we can maintain consistency.

bretlorimore committed:
Changes re: pull #10
Fix tooltip bug when playing
Move get_local_time_from_date to EclipseSimulator object definition
